pom.xml

```diff
@@ -11,7 +11,7 @@
    <version>17-SNAPSHOT</version>
    <properties>
       <version.forge>3.7.2.Final</version.forge>
-      <version.booster-catalog-service>10</version.booster-catalog-service>
+      <version.booster-catalog-service>11-SNAPSHOT</version.booster-catalog-service>
```

Comment:
Can you change that to `11`? I'll release booster-catalog before merging this PR

Reply:
I could do. about to jump on a flight, so it will be a while before I can get back to it.